examples/circle_packing.py

```diff
@@ -1,36 +1,40 @@
-import cvxpy as cp, numpy as np, cvxopt, matplotlib.pyplot as plt
-import ncvx
+import cvxpy as cp
+import numpy as np
+import matplotlib.pyplot as plt
+import ncvx as nc
 
-np.random.seed(1)
+# This example is described in Section 6.3 of the NCVX paper.
 
-N = 41 # number of circles
-r = 0.4 + 0.6 * np.random.rand(N) # radii
+rng = np.random.default_rng(seed=123)
 
-#define variables.
+N = 41                         # number of circles
+r = 0.2 + 0.3 * rng.random(N)  # radii
+
+# Define variables.
 x_vals = [cp.Variable() for i in range(N)]
 y_vals = [cp.Variable() for i in range(N)]
 
-objective = cp.Minimize(cp.maximum( *(x_vals + y_vals) ) + 0.5)
+objective = cp.Minimize(cp.maximum(*(x_vals + y_vals)) + 0.5)
 constraints = []
 for i in range(N):
-    constraints  += [0.5*r[i] <= x_vals[i],
-                     0.5*r[i] <= y_vals[i]]
+    constraints += [r[i] <= x_vals[i],
```

Comment:
What happened to the 0.5*?

Reply:
Seems like the original code was setting a circle diameter in the variable ``r``, then multiplying by 0.5 where a radius was called for.  Between renaming the variable to ``d`` or just removing the various 0.5 multiplies, the latter seemed reasonable.